tests/storage_tests.cpp

```diff
@@ -4,6 +4,49 @@
 
 using namespace sqlite_orm;
 
+#ifdef SQLITE_ORM_CTAD_SUPPORTED
+TEST_CASE("connection control") {
+    const auto openForever = GENERATE(false, true);
```

Comment:
`GENERATE` solves the issue with parallels sections tree. I have to use it as well